taipy/gui_core/_context.py

```diff
@@ -886,7 +885,7 @@ def get_job_details(self, job_id: t.Optional[JobId]):
                         [] if job.stacktrace is None else job.stacktrace,
                     )
         except Exception as e:
-            _warn(f"Access to job ({job.id if job and hasattr(job, 'id') else 'No_id'}) failed", e)
+            _warn(f"Access to job ({job_id if job_id else 'No_id'}) failed", e)
```

Comment:
... I don't really fancy this showing up for the developer.
When would job_id be None?
What about "<anonymous>"?
No big deal hu?

Reply:
`job_id or ''` ?

Reply:
Indeed. 
Here, `job_id` is a `t.Optional[JobId]`, why not just a `JobId`? 
What is the purpose of getting job details for a `None` `job_id`?

Reply:
as this method is called depending on values changing in the frontend, it can happen that job_id is not defined...

Reply:
and job_id cannot be empty/None in the exception block!